Tests/web3swiftTests/localTests/Web3ErrorTests.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  Web3ErrorTests.swift
+//
+//  Created by JeneaVranceanu on 11.11.2022.
+//
+
+import XCTest
+import Foundation
+@testable import Core
+
+class Web3ErrorTests: XCTestCase {
+
+    func test_web3ErrorReturnsExpectedDescription() {
```

Comment:
This one is a something, but anyway it's violates camelCase convention of tests naming that is used here

Reply:
Updated.